default.nix

```diff
@@ -42,7 +42,7 @@ buildGoApplication {
       --fish <($out/bin/gomod2nix completion fish) \
       --zsh <($out/bin/gomod2nix completion zsh)
   '' + ''
-    wrapProgram $out/bin/gomod2nix --prefix PATH : ${lib.makeBinPath [ go ]}
+    wrapProgram $out/bin/gomod2nix --suffix PATH : $(dirname $(which go))
```

Comment:
Still confused by this wrapping logic, `$(which go)` just find the go from current `PATH`, and we put the path back into `PATH`, what's the point of that?